### ISSUE TITLE
Psi+ (IM client) compiles fine but can't connect to TLS/SSL (complains about missing libqca-ossl)

### ISSUE THREAD

**Issue by Giova84** [haikuports/haikuports#1355]
Hi, some days ago, using HaikuPorter, I attempted to compile Psi+ (https://github.com/haikuports/haikuports/tree/master/net-im/psi_plus)

And I falls in a batch of errors (it was unable to find qmake and then QtCrypto),
after fixing/patching the code, HaikuPorter was still unable to compile Psi+, then I manually compile Psi+, and went fine: all, including plugins, properly run.

But once I started Psi+, it was unable to connect (in particular, couldn't connect to Gtalk, which is the main reason for I compiled Psi+) claiming that it cannot connect to TLS/SSL, because a plugin was missing. I will summarize: by downloading and starting an old version of Psi+ I found that such plugin is called libqca_ossl.so, and was present in the "crypto" folder of Psi+ installation's path: I was able to start this old version of Psi+ by a shell command:
QT_PLUGIN_PATH=/boot/home/config/non-packaged/psi_plus/qtplug /boot/home/config/non-packaged/psi_plus/psi-plus

But this "trick" didn't worked with the new  version which I compiled.
Obviously I compiled Psi+ using qca2 (and in facts, qca is marked as required in the Psi+'s recipe, and I didnt' see any error about qca, during compiling), but this seems not enough.

**Comment by diversys** [comment]
SSL search path is missing in current Qt 5 port. To workaround try this:

```
mkdir -p /usr/local
ln -s /boot/system/data/ssl /usr/local/ssl
```

**Comment by Giova84** [comment]
As reported in the Psi+'s recipe, it currently relies on Qt 4.8.7 (https://github.com/haikuports/haikuports/blob/master/net-im/psi_plus/psi_plus-0.16.475.recipe) so I've built it using QT 4.8.7

However I've solved for now with a very dirty workaround: I opened with DiskProbe, the libqca_ossl.so file (from the old Psi+) and I changed the ```entry i386 haiku g++-4 no-pkg-config``` to ```i386 haiku g++-4 full-config```

**Comment by korli** [comment]
@diversys any ideas where to fix the SSL search path in Qt5?

**Comment by diversys** [comment]
3dEyes is working on some fixes to Qt5 at the moment including this one!

**Comment by threedeyes** [comment]
Fixed in #294271542257e195573ceb57ef2aee510c9d89e2
